phys2denoise/cli/run.py

```diff
@@ -0,0 +1,196 @@
+                             'inputs: bts and win.',
+                        default=False)
+    metric.add_argument('-env', '--env',
+                        dest='metric_list',
+                        action='append_const',
+                        const='env',
+                        help='Respiratory pattern variability calculated across a sliding '
+                             'window. Needs the following inputs: bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rv', '--rv',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rv',
+                        help='Respiratory variance. Needs the following inputs: '
+                             'bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rvt', '--rvt',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rvt',
+                        help='Respiratory volume-per-time. Needs the following inputs: '
+                             'bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rrf', '--rrf',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rrf',
+                        help='Respiratory response function. Needs the following inputs: '
+                             'sr, os, tl, onset and tr.',
+                        default=False)
+    metric.add_argument('-rcard', '--retroicor-card',
+                        dest='metric_list',
+                        action='append_const',
+                        const='r_card',
+                        help='Computes regressors for cardiac signal. Needs the following '
+                             'inputs: tr, nscans, slt and n_harm.',
+                        default=False)
+    metric.add_argument('-rresp', '--retroicor-resp',
+                        dest='metric_list',
+                        action='append_const',
+                        const='r_resp',
+                        help='Computes regressors for respiratory signal. Needs the following  '
+                             'inputs: tr, nscans, slt and n_harm.',
+                        default=False)
+    optional.add_argument('-outdir', '--output-dir',
+                          dest='outdir',
+                          type=str,
+                          help='Folder where output should be placed. '
+                               'Default is current folder.',
+                          default='.')
+    optional.add_argument('-sr', '--sample-rate',
+                          dest='sample_rate',
+                          type=float,
+                          help='Sampling rate of the physiological data in Hz.',
+                          default=None)
```

Comment:
I thought this workflow was BIDS-specific? If so, I don't think you need to provide information that must be stored in the metadata.

Reply:
ATM is not necessarily BIDS-specific, so we'll still need more info from the user (but hey, if it's BIDS, it will be automated - that's why they are optional flags)

Reply:
In fact, if it was the peakdet saved file, we could get the same information from it, but...